packages/io/__tests__/io.test.ts

```diff
@@ -331,11 +332,26 @@ describe('rmRF', () => {
     await fs.appendFile(filePath, 'some data')
     await assertExists(filePath)
 
-    const fd = await fs.open(filePath, 'r')
-    await io.rmRF(testPath)
-
-    await assertNotExists(testPath)
+    // for windows, we need to explicitly set an exlcusive lock
+    // or we won't get an EBUSY error. this can be fixed after
+    // https://github.com/libuv/libuv/issues/3267 is resolved
+    // exclusive lock windows flag: https://github.com/nodejs/node/blob/c2e4b1fa9ad0b744616c4e4c13a5017772a630c4/deps/uv/src/win/fs.c#L499-L513
+    const fd = await fs.open(
+      filePath,
+      fs.constants.O_RDONLY | ioUtil.UV_FS_O_EXLOCK
+    )
+    if (ioUtil.IS_WINDOWS) {
+      try {
+        // additionally, can't stat an open file on Windows without getting EPERM
+        await io.rmRF(testPath)
```

Comment:
you're expecting an exception to be thrown, right?  if so, we probably want to fail the test on the line immediately following the `io.rmRM(testPath)` call since we expect anything else within the `try` block to be unreachable code.